tests/unit/type/stream-cgroups.tcl

```diff
@@ -543,6 +543,25 @@ start_server {
         assert_equal [s total_error_replies] 1
     }
 
+    test {XGROUP DESTROY removes all consumer group references} {
```

Comment:
if I'm not mistaken, XTRIM test with new options is missing. Does it make sense to add it just for coverage or maybe we hit same underlying functions enough from other commands?

Reply:
aha, yes, `XTRIM` and `XADD` share this trim code.